schema.sql

```diff
@@ -13,3 +13,22 @@ CREATE TABLE animals(
 );
 
 ALTER TABLE animals ADD species varchar(255);
+
+
+-- Query Multiple Tables
+create table owners(
+    id int serial primary key,
+    full_name varchar(255),
+    age int
+)
+
+create table species(
+    id int serial primary key,
+    name varchar(255)
+)
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/172025676-a0d520d4-05f9-4e52-bb8e-380bc824939d.png)- 

- [ ] Kindly note that you are having a syntax error here, you don't need to specify the type of the id(int) when you are using `serial`. Hint :point_down: 
 
```sql
create table owners(
    id  serial primary key,
    full_name varchar(255),
    age int
);
```

- [ ] Also make sure you are sending a command(query) with a `;`